ElrondNetwork/api.elrond.com
# filter nfts by nonce before after

## Proposed Changes
-  add nonce filter

## How to test
REST API
/nfts:
- nfts?nonce=200 -> should return all NFTS/SFTS that contains given nonce
- nfts/count?nonce=300 -> should return count 516
- nfts/c?nonce=300 -> should return count 516 

/collections/:collection/nfts
- collections/ANGELSLOGO-4adf87/nfts?nonce=8 -> should return ANGELSLOGO-4adf87-08 NFT with nonce = 8
- collections/BACPASS-04e709/nfts?nonceBefore=10 -> should return all NFTs from specified collection in descending order

API GraphQL
- create nfts query and add nonce: 200 as input
